src/main/pages/user-management/user-management.adoc

```diff
@@ -8,67 +8,102 @@ folder: user-management
 ---
 
 
-[id="auth-and-user-management"]
-== Auth and User Management
+[id="authorization-and-user-management"]
+== Authorization and user management
 
-Eclipse Che relies on http://www.Keycloak.org[Keycloak] to create, import, manage, delete and authenticate users. Keycloak uses its own authentication mechanisms and user storage. Eclipse Che requires a Keycloak token when access to Che resources is requested.
+Eclipse Che uses http://www.Keycloak.org[Keycloak] to create, import, manage, delete, and authenticate users.  Keycloak uses built-in authentication mechanisms and user storage.  It can use a third-party identity management systems to create and authenticate users. Eclipse Che requires a Keycloak token when you request access to Che resources.
```

Comment:
"It can use third-party identity management systems to create and authenticate users."